internal/cli/agent.go

```diff
@@ -18,6 +18,7 @@ package cli
 
 import (
 	"fmt"
+	"github.com/yorkie-team/yorkie/yorkie/log"
```

Comment:
Our import ordering is below.

- Standard library
- Third party library
- Yorkie

GoFmt helps you sort by each group. It's not perfect, so some corrections are needed.

Reply:
I will follow this project style guide :)
Are there any scripts for checking this format is proper or not before commit?

Reply:
You can check the code style with `make lint`, but it doesn't check the import order very well. 😭